src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -182,35 +182,32 @@ function bindCopyButton() {
         return false;
     });
 }
+var prevRow;
 
 function bindCopyEvents() {
-
-    $('#copyTableModal >tbody>tr').on('click', function(e) {
+    $('#copyTableModal > tbody > tr').on('click', function(e){
         e.preventDefault();
-
+        
         var currentRow = $(this);
```

Comment:
a jQuery wrapped object should have a $ prefix so it is immediately obvious that it is a jQuery wrapped object and not a plain JavaScript object.